pages/style-guide/prose.md

```diff
@@ -0,0 +1,113 @@
+
+### Code Blocks
+
+Code blocks should be set off with triple backticks and should **not** be indented. (That is to say, the site prefers the use of "fenced blocks" notation supported by [gfm](http://github.github.com/github-flavored-markdown/).)
+
+## Writing Style
+
+Content should be educational and accessible to a broad audience of developers. The primary target audience is beginning to intermediate jQuery users, with advanced jQuery users as a secondary audience. When creating content, please keep one of these audiences in mind as well as the following style guidelines:
+
+### Prose & Grammar
+
+- Use the Oxford comma in a list of three or more items:
+	- **Yes:** The `load`, `scroll`, and `error` events (e.g. on an `<img>` element) do not bubble.
+	- **No:** The `load`, `scroll` and `error` events (e.g. on an `<img>` element) do not bubble.
+- Numbers:
+	- Spell out numbers below 10 (e.g. one, two, three). 
+	- Use numerals for numbers 10 and above (e.g. 10, 20, 100).
+- Abbreviations:
+	- Spell out abbreviated words on first reference, followed by the abbreviation in parentheses. Use abbreviations on second reference.
+
+#### Code Within Prose
+
+- Always use a `code` tag to denote code from prose.
+- Properties: use a dot, followed by the property name, e.g. `.length`.
+- Functions: use the function name, followed by parentheses, e.g. `myfunction()`.
+- Methods: use a dot, followed by the method name, followed by parentheses, e.g. The `.focus()` method is a shortcut for  `.bind( "focus", handler )` in the first and second variations, and `.trigger( "focus" )` in the third.
+
+#### Article & Sentence Structure
+
+- Use headings to break up content into easier-to-read sections. Begin headings within an article with `<h2>`.
+- Keep sentences short and to the point. A good rule-of-thumb is to break up any sentence longer than 21 words into two or more seperate thoughts.
+- Lists:
+	- Use bulleted lists when necessary to share a series of five or more points.
+	- Use numbered lists only when providing step-by-step instruction – note that this should be avoided.
+	- Use a period at the end of each ordered list item, and a period or comma at the end of an unordered list item.
+
+#### Spelling
+
+- Use standard American English spelling.
+- Capitalization:
+	- Capitalize all proper nouns.
+	- Do not capitalize HTML elements in code examples.
+	- Capitalize all words in a heading or sub-heading with the exception of article adjectives and the prepositions like "with," "of," or "to."
+	- Capitalize the first word in a list.
+- Punctuation:
+	- Periods and commas go inside quotation marks.
+	- Avoid using semicolons.
+
+#### Pronoun Usage
+
+- Don't use "I," "me," "us," "our," "we," and gender-specific pronouns such as "him" or "she."
```

Comment:
@scottgonzalez Do you wish to have the word "or" replaced by the word "and" both in line 67 and line 75?